docs/how-tos/train-deploy-ml.md

```diff
@@ -306,27 +331,42 @@ To expand the image, click on the expand side menu arrow in the corner of the im
 {{<gif webm_src="/services/data/label-dog-big.webm" mp4_src="/services/data/label-dog-big.mp4" alt="Add a bounding box around the dog in an image in a big menu">}}
 
 Repeat this with all images.
-To see all the images that have bounding boxes, you can filter your dataset by selecting the label from the **Bounding box labels** dropdown in the **Filters** menu.
 
+You can add one or more bounding boxes for objects in each image.
+{{< /expand >}}
+
+Determining a descriptive state about an image.
```

Comment:
```suggestion
- Classifying an image as a whole.
  In other words, determining a descriptive state about an image.
```
Or something—I think we need to use the word classify here in some way to connect it to other bits of the documentation where we usually talk about classifiers